src/components/utils/blocks.ts

```diff
@@ -119,3 +137,98 @@ export function convertStringToBlockData(stringToImport: string, conversionConfi
     return {};
   }
 }
+
+/**
+ * Returns list of all available conversion menu items
+ *
+ * @param currentBlock - current block we need to get convetion data for
+ * @param editorModules - access to editor modules
```

Comment:
I think, passing Editor Modules to util is a bad practice. Util should be decoupled from other modules

Reply:
Agree, would be better to pass the public API instead

Reply:
fixed